src/app/modules/angular-slickgrid/services/__tests__/bsDropdown.service.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { BsDropDownService } from '../bsDropdown.service';
+
+describe('bsdropdown-service', () => {
+
+  // this test should be written
+  describe('Render method', () => {
+    const service = BsDropDownService;
+    it('should show check is the dropdown is displayed after clic', () => {
+      expect(true).toBe(true);
```

Comment:
I hope you will eventually add more tests than this, because I won't accept this PR as it is. Please provide sufficient Jest Unit Tests before I can review and accept this PR